extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -71,16 +75,15 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 		}
 
 		function startBtfQueue() {
-			var context = adContext.getContext(),
-				roadblockBlocksBtf = uapContext.isRoadblockLoaded() && win.ads.runtime.disableBtf;
+			var context = adContext.getContext();
 
 			log('startBtfQueue', log.levels.info.debug, logGroup);
 
 			if (btfQueueStarted) {
 				return;
 			}
 
-			if (context.opts.premiumAdLayoutEnabled && !roadblockBlocksBtf) {
+			if (context.opts.premiumAdLayoutEnabled && !isBTFDisabledByCreative()) {
```

Comment:
Can we, therefore, remove `uapContext.isRoadblockLoaded()`?
 (https://github.com/Wikia/app/blob/9621f1a95c03f77d3910f6cd054351d0384220c5/extensions/wikia/AdEngine/js/context/uapContext.js#L39)

Reply:
And was this the actual bug? Because everything else looks cosmetic + unit tests :+1: :)

Reply:
I believe this change breaks kill code in roadblock. By design it should block all slots except `TOP_RIGHT_BOXAD`. In this case all premium slots will be unblocked.

Reply:
Ups, I was wrong :)